libsrc/c64/soft80_conio.s

```diff
@@ -0,0 +1,230 @@
+;
+; Low level stuff for screen output/console input
+;
+
+        .constructor    soft80_init, 24
```

Comment:
Okay, I removed the explicit prios from both PCE constructors (and added them to the Wiki page mantioned).